tox.ini

```diff
@@ -1,5 +1,5 @@
 [tox]
-envlist = py25,py26,py27,pypy,py32,py33,py34,lint
```

Comment:
We still support 2.5 so please add this back and you can then explicitly specify which environments to use in the Dockerfile - `CMD tox -e py26,py27,pypy,py32,py33,py34,lint`.